src/clj/puppetlabs/services/ca/certificate_authority_core.clj

```diff
@@ -3,16 +3,15 @@
   (:require [puppetlabs.puppetserver.certificate-authority :as ca]
             [puppetlabs.puppetserver.ringutils :as ringutils]
             [puppetlabs.puppetserver.liberator-utils :as utils]
+            [puppetlabs.comidi :as comidi :refer [GET ANY PUT]]
             [slingshot.slingshot :as sling]
             [clojure.tools.logging :as log]
             [clojure.java.io :as io]
             [clojure.string :as string]
             [schema.core :as schema]
             [cheshire.core :as cheshire]
-            [compojure.core :as compojure :refer [GET ANY PUT]]
             [liberator.core :refer [defresource]]
             [liberator.representation :as representation]
-            [liberator.dev :as liberator-dev]
```

Comment:
In another ns, this was left in but commented out... would be nice to be consistent about that?